py3status/modules/spaceapi.py

```diff
@@ -56,6 +61,7 @@ def check(self):
             reader = codecs.getreader("utf-8")
             data = json.load(reader(json_file))
             json_file.close()
+            self.url = data['url']
```

Comment:
It wouldn't be well-formed in the spaceapi standard, but you're right; better safe than sorry. Changed.